apps/nestjs-api/src/tp-job-listings/tp-job-listings.service.ts

```diff
@@ -41,6 +42,7 @@ export class TpJobListingsService {
       ['Account__r.ReDI_Visible_to_Jobseekers__c']: true,
       ['Account__r.ReDI_Talent_Pool_State__c']:
         CompanyTalentPoolState.PROFILE_APPROVED,
+      Status__c: TpJobListingStatus.ACTIVE,
```

Comment:
We'll need to put in place a mechanism in Salesforce to set status to EXPIRED once the clock ticks past the expiry date.

Where can I review that? I suppose it'll go live at the same time this goes live.

Depending on how confident we are in the reliability of that mechanism, we might want to add a "expires at date is in the past" filter here. How do we feel about that?